EcalTiming/plugins/EcalTimingCalibProducer.cc

```diff
@@ -505,9 +519,10 @@ EcalTimingCalibProducer::Status EcalTimingCalibProducer::duringLoop(const edm::E
 #endif
 	// If we got less than the minimum recHits, continue
 	if(eventTimeMap_.size() < _recHitMin) return kContinue;
-	if(timeEB.num() < 4) return kContinue;
-	if(timeEEM.num() < 4 && timeEEP.num() < 4) return kContinue;
+	if(timeEB.num() + timeEEM.num() + timeEEP.num() < _minEntries) return kContinue;
```

Comment:
I will remove the second line if you agree.

-- 
 							Shervin


On Mon, 15 Jun 2015, Emanuele Di Marco wrote:

> Date: Mon, 15 Jun 2015 09:37:38 +0200
> From: Emanuele Di Marco <notifications@github.com>
> Reply-To: ECALELFS/EcalTiming
>     <reply+0045c225ff32d7fda758b4d61938e5bc6ac67c2c1b3e62d192cf00000001119642c
>     292a170ce01ee56c2@reply.github.com>
> To: ECALELFS/EcalTiming <EcalTiming@noreply.github.com>
> Cc: shervin86 <shervin@cern.ch>
> Subject: Re: [EcalTiming] Timing 744 (#3)
> 
>> @@ -505,9 +519,10 @@ EcalTimingCalibProducer::Status EcalTimingCalibProducer::duringLoop(const edm::E
>>  #endif
>>  	// If we got less than the minimum recHits, continue
>>  	if(eventTimeMap_.size() < _recHitMin) return kContinue;
>> -	if(timeEB.num() < 4) return kContinue;
>> -	if(timeEEM.num() < 4 && timeEEP.num() < 4) return kContinue;
>> +	if(timeEB.num() + timeEEM.num() + timeEEP.num() < _minEntries) return kContinue;
>
> This is originally from Peter. I think one is the minimum number of rechits in the collection, the second is the minimum number of entries in the final map to make the averages (i.e. after the selections).
>
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/ECALELFS/EcalTiming/pull/3/files#r32396994
>